src/scenes/Activities/ActivityHistory/store.js

```diff
@@ -158,12 +155,13 @@ export default class {
    * @return {string} URL path for the newest Oracle.
    */
   @action
-  getOracleAddress = async (topicAddress) => {
-    if (topicAddress) {
-      const filters = [{ topicAddress }];
-      const order = { field: 'endTime', direction: SortBy.DESCENDING };
-      const { oracles } = await queryAllOracles(this.app, filters, order);
-      const path = getDetailPagePath(oracles);
+  getEventAddress = async (address) => {
```

Comment:
this func should not be necessary anymore. events are a single contract instead of multiple topic/oracle contracts. all of the urls for events should be simple now, `/event/{eventAddress}`, and should never change